### ISSUE TITLE
Cluster import yaml has incorrect document break end of file

### ISSUE THREAD

**Issue by ryanelliottsmith** [rancher/rancher#31252]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
Download cluster import yaml from Rancher
Try to apply to a new cluster with [kubernetes-python client](https://github.com/kubernetes-client/python)

**Result:**
Client fails to parse the yaml doc with error:
```
File "/usr/local/lib/python3.6/site-packages/kubernetes/utils/create_from_yaml.py", line 70, in create_from_yaml
if "List" in yml_document["kind"]:
TypeError: 'NoneType' object is not subscriptable
```

**Other details that may be helpful:**
This appears to be because the import yaml has a document break (`---`) at the end of the file, which indicates a new doc

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.4.x > v2.5.3

gz#14869

**Comment by kravciak** [comment]
I reproduced this on `v2.5.6-rc4` and fix works fine `v2.5-head` and `master-head`.

**Comment by kravciak** [comment]
To verify I did those steps on 

Visual check:
- Open rancher global cluster view
- Click `Add Cluster -> Other Cluster -> Create`
- Open yaml file url from kubectl command in browser and check it does not end with `---`

Cluster import check:
- Download import yaml file locally into `/home/kravciak/import.yaml`
- Create kubernetes cluster from google cloud UI
- Run Import script from python kubernetes client

```python
from kubernetes import client, config, utils

def main():
    # generated by gcloud command
    config.load_kube_config()
    k8s_client = client.ApiClient()

    # import.yaml downloaded from rancher url
    yaml_file = '/home/kravciak/import.yaml'
    utils.create_from_yaml(k8s_client, yaml_file)

if __name__ == '__main__':
    main()
```
